docs/build_reference.ps1

```diff
@@ -0,0 +1,33 @@
+# If stdlib-reference folder does not exist, clone from github repo
+if (-not (Test-Path ".\stdlib-reference")) {
+    git clone https://github.com/shader-slang/stdlib-reference/
+}
+else {
+    cd stdlib-reference
+    git pull
+    cd ../
+}
+Remove-Item -Path ".\stdlib-reference\global-decls" -Recurse -Force
+Remove-Item -Path ".\stdlib-reference\interfaces" -Recurse -Force
+Remove-Item -Path ".\stdlib-reference\types" -Recurse -Force
+
+git describe --tags | Out-File -FilePath ".\stdlib-reference\_includes\version.inc" -Encoding ASCII
+
+cd stdlib-reference
+& ../../build/Release/bin/slangc -compile-stdlib -doc
+Move-Item -Path ".\toc.html" -Destination ".\_includes\stdlib-reference-toc.html" -Force
+git config user.email "bot@shader-slang.com"
+git config user.name "Stdlib Reference Bot"
+git add .
```

Comment:
Can we specify the exact file names need to be added to avoid unexpected problems?

Reply:
Updated the comments on this.